.github/workflows/push.yml

```
@@ -200,8 +200,13 @@ jobs:
         uses: ./.github/actions/load-deps
 
       - name: test-everything
+        id: test-everything
         uses: ./.github/actions/test-everything
 
+      - name: Set test-everything env
+        run: |
```

actionlint (1.7.4)

property "test_everything" is not defined in object type {success: string}

(expression)